docs/index.rst

```diff
@@ -0,0 +1,49 @@
+REISE.jl
+========
+REISE.jl (Renewable Energy Integration Simulation Engine) is the simulation engine
+developed by BES to run power flow study in the U.S. electric grid. REISE.jl is an
+open source package written in Julia that is available on `GitHub
```

Comment:
Should "power flow study" be "power-flow studies"?

I believe there should be a hyphen between "open" and "source"

Reply:
Done